packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.types.ts

```diff
@@ -264,6 +264,17 @@ export interface IContextualMenuProps extends IBaseProps<IContextualMenu>, IWith
   delayUpdateFocusOnHover?: boolean;
 }
 
+/**
+ * {@docCategory ContextualMenu}
+ */
+export interface IInternalContextualMenuItem extends IContextualMenuItem {
```

Comment:
It's weird to export a type that says "internal". How about calling it `IContextualMenuItemRenderArgs`?

Reply:
My only concern is the fact that it's just an extension of IContextualMenuItem so it still is a menu item. Adding RenderArgs might make it look more like a props object instead.

Regardless, I like it better than IInternalContextualMenuItem so I'll update this

Reply:
I think that IConextualMenuItemRender is definitely better, I might say it should be `IContextualMenuItemRenderProps` to keep things consistent with our naming conventions.